ga4gh/client.py

```diff
@@ -827,8 +799,7 @@ def __init__(self, backend):
             "individuals": self._backend.runGetIndividual,
             "rnaquantificationsets": self._backend.runGetRnaQuantificationSet,
             "rnaquantifications": self._backend.runGetRnaQuantification,
-            "expressionlevels": self._backend.runGetExpressionLevel,
-            "featuregroups": self._backend.runGetFeatureGroup
+            "expressionlevels": self._backend.runGetExpressionLevel
```

Comment:
nitpick: it's good to keep the comma at the end of the list so only one line is changed in the diff, not 2 (and in a future diff where a line is added to this list for the same reason)